core/mutator/entry/mutator.go

```diff
@@ -90,14 +89,14 @@ func (*Mutator) Mutate(oldValue, update proto.Message) (proto.Message, error) {
 
 	kv := *updated
 	kv.Signatures = nil
-	if err := verifyKeys(oldEntry.GetAuthorizedKeys(),
+	if err := verifyKeys(oldEntry.GetValue().GetAuthorizedKeys(),
 		newEntry.GetAuthorizedKeys(),
 		kv,
 		updated.GetSignatures()); err != nil {
```

Comment:
Not your diff, but this `; err != nil{` check is really lost here.  Consider a line break just before the `)` immediately preceding it.

Reply:
Good idea.